coremltools/converters/mil/frontend/torch/test/test_torch_conversion_api.py

```diff
@@ -48,12 +48,7 @@
     torch.manual_seed(1818)
 
 if _HAS_EXECUTORCH:
-    from executorch import exir
-
-    _CAPTURE_CONFIG = exir.CaptureConfig(enable_aot=True)
-    _EDGE_COMPILE_CONFIG = exir.EdgeCompileConfig(
-        _check_ir_validity=False,
-    )
+    import executorch.exir
```

Comment:
this import is a little bit confusing,
I think we could just `import executorch`?

Reply:
`executorch` has some issue with their importing mechanism 😞 Somehow we have to import `executorch.exir`

Reply:
Okay!